### PR TITLE
CFE-3267/master: Added Code of Conduct

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

File: CODE_OF_CONDUCT.org [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Examples of unacceptable behavior by participants include:
+
+- The use of sexualized language or imagery
+- Personal attacks
+- Trolling or insulting/derogatory comments
+- Public or private harassment
+- Publishing other's private information, such as physical or electronic
+  addresses, without explicit permission
+- Other unethical or unprofessional conduct.
```
Comment (olehermanse) [inline CODE_OF_CONDUCT.org:23]: Remove period since none of the other bullets have it.

**Comment by nickanderson** [comment]
As you wish.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+maintainers commit themselves to fairly and consistently applying these
+principles to every aspect of managing this project. Project maintainers who do
+not follow or enforce the Code of Conduct may be permanently removed from the
+project team.
+
+This code of conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting a Member project maintainer or <contact@cfengine.com>.
```
Comment (craigcomstock) [inline CODE_OF_CONDUCT.md:35]: not sure why Member is capitalized or why it is there exactly. "may be reported by contacting a project maintainer or <contact@cfengine.com>." is good enough?

Comment (nickanderson) [reply CODE_OF_CONDUCT.md:35]: @michaelclelland  ^ ?

Comment (olehermanse) [reply CODE_OF_CONDUCT.md:35]: Was it perhaps supposed to be `Mender project maintainer`?

**Pull request merged**